storage/common.go

```diff
@@ -67,6 +67,16 @@ type KeyValueStore interface {
 	Close() error
 }
 
+//DistQueue is an interface describing queue stores like redis
+type DistQueue interface {
```

Comment:
Just `Queue` should be sufficient.

Reply:
I was thinking about this... the existing RepairQueue implementation uses a mutex to make itself locally thread-safe.  Redis should already be thread-safe, so this locking seems like a bad idea.  Maybe I should rename to Queue, alias to DistQueue, as well as remove the mutex from RepairQueue?

Reply:
I don't think I'm following. I mean it still should be thread-safe as other storage implementation. I just don't think the `Dist` should be part of the type name. Adding comment `A threadsafe FIFO queue.` should suffice.

Reply:
Renamed